.eslintrc

```diff
@@ -67,6 +67,10 @@
     "worker": true
   },
 
+  "globals": {
+    "JsSpeechRecognizer"
```

Comment:
You don't need to make it a global if it's only loaded via es6 modules in a file.